website/docs/source/docs/networking/private_network.html.md

```diff
@@ -12,7 +12,7 @@ Private networking by the Parallels provider is fully compatible with the basic
 Vagrant approach.
 
 In order to implement a private network, the Parallels provider configures the
-internal [Host-Only](http://download.parallels.com/desktop/v14/docs/en_US/Parallels%20Desktop%20User's%20Guide/33018.htm)
+internal [Host-Only](https://download.parallels.com/desktop/v14/docs/en_US/Parallels%20Desktop%20User's%20Guide/33018.htm)
```

Comment:
v16 here as well?

Reply:
I agree! I didn't see that before. I updated the pull request